### PR TITLE
Changed default Polish filters list to secured (https) version.

### PR DESCRIPTION
**Pull request by Piter432**
For non-secure (http) version add this -> http://certyficate.it/adblock/adblock.txt <- to custom uBlock Origin's filters list.

### CODE REVIEW

File: assets/ublock/filter-lists.json [hunk 1]
```diff
@@ -144,11 +144,12 @@
         "title": "Basic tracking list by Disconnect",
         "group": "privacy"
         },
-    "http://www.certyficate.it/adblock/adblock.txt": {
+    "https://www.certyficate.it/adblock/adblock.txt": {
         "off": true,
         "title": "POL: Adblock polskie reguly",
         "group": "regions",
         "oldLocation": "raw.githubusercontent.com/adblockpolska/Adblock_PL_List/master/adblock_polska.txt"
```
Comment (krzysztof-sikorski) [inline assets/ublock/filter-lists.json]: Missing comma.

Comment (Piter432) [inline assets/ublock/filter-lists.json]: @krzysztof-sikorski Fixed. Thank you for reporting this :)

**Comment by Piter432** [comment]
@gorhill Can you merge this, please? :)

**Comment by gorhill** [comment]
Unecessary, because of #602: all 3rd-party filter lists which are not seleted by default are now external. The [mapping](https://github.com/gorhill/uBlock/blob/master/src/js/background.js#L143) uses the `https` version.

**Pull request closed without merging**